rollup.config.ts

```diff
@@ -14,6 +14,7 @@ export default [
         format: 'cjs',
       },
     ],
+    external: [/^lighthouse/],
```

Comment:
probably need to exclude Lighthouse in the next definition too?

Reply:
I tried that, I don't think it changes anything